### ISSUE TITLE
[Keyboard Navigation - WIX - Setup Successful]"Close" button is not accessible via keyboard on "Setup successful" window.

### ISSUE THREAD

**Issue by sikoppar** [wixtoolset/issues#6037]
* Which version of WiX are you building with?
> 3.7.1224.1

* Which version of Visual Studio are you building with (if any)?
> Visual Studio 2017 (v141)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
>0.9.21.62588

* Which version of .NET are you building with?
> Not Applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
>Not Applicable

* Describe the problem and the steps to reproduce it.
User Experience:

If, any control is not accessible by the keyboard, then the users who depends upon keyboard for navigation might get affected and may not be able to use the complete functionality of the application.

Pre Requist:
1. Download the installer from https://www.microsoft.com/en-us/download/details.aspx?id=54688
2. Navigate to NPS Extension for Azure MFA Setup and activate it by using Enter key.

Repro Steps:
1. Navigate to "Install" button of "NPS Extension for Azure MFA " popup window and select it.
2. Navigate to "Windows(OS)" "Do you want to Allow this app "popup" window.
3. Navigate to "Yes" button and select it.
4. Navigate to "Setup Progress" Popup window and wait till setup completed.
5. Navigate to all the controls of "Setup Successful" popup window.
6. Verify if , Close button is accessible via keyboard or not.

Actual Result:
"Close" button is not accessible via keyboard on "Setup successful" window. 

Expected Result:
"Close" button should be accessible via keyboard on "Setup successful" window.

**Comment by BMurri** [comment]
That version of WiX is over 5 years old, and there have been changes to the WixStdBa since then. Do these issues replicate with v3.11.1?

**Comment by AlasdairKing** [comment]
Can't replicate. Windows 11 Pro. 
1. Run installation as described above. 
2. On final screen, expect "Close" button to have have focus (and therefore be "accessible via keyboard")
3. Observe that the "Close" button does indeed have the focus, and pressing the Return key on the keyboard does indeed close the installer window.

I'm not clear what @sikoppar means by "accessible via keyboard" - in most contexts this would mean "you can tab to the UI element" but the Close button is the only control that takes focus on the window, so you can't tab to it. But it starts with focus, so that's not what it means here!

**Comment by barnson** [comment]
Yeah, drive-by accessibility bugs are like this. Since 2019, the team switched to WiX v5.
